Orange/widgets/visualize/owmosaic.py

```diff
@@ -482,6 +479,7 @@ def reset_graph(self):
 
     def set_color_data(self):
         if self.data is None or len(self.data) < 2 or len(self.data.domain.attributes) < 1:
+            self.coloring_changed()
```

Comment:
Is this necessary? It would stop vizrank, which can't be running in this case anyway, and it updates graph, which is also unnecessary.

Reply:
Removed.